### ISSUE TITLE
Поддержка kodeco.com

### ISSUE THREAD

**Issue by Wizzmo** [ilyhalight/voice-over-translation#702]
### Describe your feature / Опишите ваше предложение

Если есть возможность, добавьте, пожалуйста, поддержку сайта kodeco.com, где выкладывают одни из самых лучших курсов по моб. программированию (swift, kotlin. flatter)

**Comment by ilyhalight** [comment]
Там используется вимео, который считает видео встроенные на такие сайты как kodeco приватными, поэтому их не получится перевести, если не найдется какой-то обходной способ

**Comment by Wizzmo** [comment]
@ilyhalight а не подскажите, какой сейчас самый простой способ делать перевод видео, которые хранятся локально, чтобы их не приходилось куда-то заливать?
Пробовал через прямые веб-ссылки на .mp4 - не работает.
Пробовал запускать через локальное видео с компа - не работает (но вроде и не должно работать)
Пробовал через Яндекс Диск - не работает. Для работы необходимо добавить скрипт в CSP. В Инструкции сказано в настройках Tampermonkey в списке "Добавить TM в политику безопасности CSP: " установите значение "Да". А по факту там нет такого значения.
<img width="647" alt="Monosnap Monosnap 2024-07-25 15-00-34" src="https://github.com/user-attachments/assets/cc96247c-f587-4d95-a769-df4d263643a7">

**Comment by ilyhalight** [comment]
> . В Инструкции сказано в настройках Tampermonkey в списке "Добавить TM в политику безопасности CSP: " установите значение "Да". А по факту там нет такого значения.

Так, скорее всего, только с Tampermonkey MV3. Нужно использовать какое-то стороннее расширение для отключения CSP в Яндекс диске, либо установить "Удалить полностью" (не рекомендуется)

> Пробовал через прямые веб-ссылки на .mp4

Что пишет?

> а не подскажите, какой сейчас самый простой способ делать перевод видео, которые хранятся локально, чтобы их не приходилось куда-то заливать?

Можно попробовать ВК Видео, ВК, Гугл диск (<40 мин.), либо какой-нибудь S3

**Comment by Wizzmo** [comment]
@ilyhalight спасибо! Получилось включить перевод видео, которые лежат на Яндекс Диске. Установил "Удалить полностью" и заработало.
